### PR TITLE
[Python API] Rename Variant  to OVAny

### PR DESCRIPTION
**Pull request by artkuli**

### CODE REVIEW

File: src/bindings/python/src/openvino/runtime/__init__.py [hunk 1]
```diff
@@ -54,7 +54,7 @@
 from openvino.runtime.ie_api import ExecutableNetwork
 from openvino.runtime.ie_api import InferRequest
 from openvino.runtime.ie_api import AsyncInferQueue
-from openvino.runtime.ie_api import Variant
+from openvino.runtime.ie_api import OVAny
```
Comment (ilya-lavrenov) [inline src/bindings/python/src/openvino/runtime/__init__.py:57]: why not `Any`?

**Review by tomdol — APPROVED** [review]

**Review by jiwaszki — COMMENTED** [review]
🚀

**Review by jiwaszki — APPROVED** [review]

**Comment by tomdol** [comment]
@ilyachur @ilya-lavrenov @akuporos - what do you think about the new name? ov::Any cannot be exposed to Python as Any because that's a reserved keyword. Does `OVAny` sound good in this case?

**Comment by ilya-lavrenov** [comment]
> Does OVAny sound good in this case?

For me - yes. I cannot suggest something smarter

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**